v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/constants/SourceDbToSpannerConstants.java

```diff
@@ -31,4 +31,7 @@ public class SourceDbToSpannerConstants {
 
   /* The tag for rows which were filtered based on custom transformation response. */
   public static final TupleTag<RowContext> FILTERED_EVENT_TAG = new TupleTag<>();
+
+  /* Misc constants */
```

Comment:
Add some context about this constant.

Also rename it to MAX_RECOMMENDED_TABLES_PER_JOB